### DISCUSSION TITLE
Why are there different values between data.M and data.Ycrb ?

### DISCUSSION THREAD

**Discussion by hxrkpib** [stack-of-tasks/pinocchio#1982] [Q&A] [answered]
Hello,
I have a confuse when calculating the inertia. I load model from a urdf file, then run bellow code:
```
pinocchio::crba(*model_floating_, *data_floating_, q);
data_floating_->M.triangularView<Eigen::StrictlyLower>() = data_floating_->M.transpose().triangularView<Eigen::StrictlyLower>();
std::cout << "M" << std::endl << data_floating_->M.topLeftCorner(6, 6) << std::endl;
std::cout << "ycrb1" << std::endl << data_floating_->Ycrb[1] << std::endl;
```
I get common mass value, but there is inertia value bias between  data.M and data.Ycrb.
![image](https://github.com/stack-of-tasks/pinocchio/assets/28836328/cae1778e-94e2-479c-85f9-8e9fa0bd253d)

**Comment by jcarpent** [comment, marked as answer]
The inertia quantity in the spatial inertia is expressed around the center of mass, while it is expressed in the local joint frame for the mass matrix.
